### ISSUE TITLE
Fomm and Motion-cos

### ISSUE THREAD

**Issue by newExplore-hash** [AliaksandrSiarohin/motion-cosegmentation#42]
hi, aliaksandrSiarohin, very appreciate your work, i find keypoint detector network of fomm is similar with segmentation module of motion cos, and others (generator and dense motion) are same totally, when i train the fomm model from scrath on new dataset, Can this model be used to inference in motion-cos directly？or do some parameters need to be modified?

**Comment by AliaksandrSiarohin** [comment]
Hi. You mean for part swapping? No for part swapping you need segmentation. Fomm does not provide you with the segmentation.

**Comment by newExplore-hash** [comment]
yeah, is part swapping,  as for segmentation, i can use face parser for segmentation, My question is whether the fomm model can be directly used for part-swap inference when i use face parser for segmentation.

**Comment by AliaksandrSiarohin** [comment]
Yes you can use it in supervised mode.
